bundles/org.openhab.binding.velux/src/main/resources/OH-INF/i18n/velux.properties

```diff
@@ -133,6 +133,8 @@ channel-type.velux.scenes.description = Scenes which are configured on the Bridg
 channel-type.velux.check.label = Check of configuration
 channel-type.velux.check.description = Result of the check of current item configuration.
 #
+channel-type.velux.windowPosition.label = Position
```

Comment:
Maybe "Window Position" would be better ?

Reply:
I am going to merge to have your PR included in coming milestone.
You could change that in a next PR if you like.

Reply:
Done.